enterprise/server/backends/pubsub/BUILD

```diff
@@ -6,6 +6,7 @@ go_library(
     importpath = "github.com/buildbuddy-io/buildbuddy/enterprise/server/backends/pubsub",
     visibility = [
         "//enterprise:__subpackages__",
+        "//server/testutil:__subpackages__",
```

Comment:
We should avoid mixing these code bases which have different licenses (MIT vs Enterprise).

Couple of options:
 - we could move pubsub into the OS repo but I think it's largely useless on its own so we'd mostly be doing it to solve this viz problem
 - we could add an "enterprise" version of GetTestEnv that calls the OS one and then injects any enterprise-specific stuff we need
 - something else...

wdyt?